File: 2655-rearrange-array-to-maximize-prefix-score/rearrange-array-to-maximize-prefix-score.cpp
class Solution {
public:
    int maxScore(vector<int>& nums) {
        long long sum = 0 ; 
        int res = 0 ;
        sort(nums.begin() , nums.end() , greater<int>());
        for(auto it : nums){
            sum += it;
            if(sum>0) res++;
        }
        return res;
    }
};

File: 2655-rearrange-array-to-maximize-prefix-score/rearrange-array-to-maximize-prefix-score.py
class Solution:
    def maxScore(self, nums: List[int]) -> int:
        nums = sorted(nums)
        nums = nums[::-1]
        res = []
        sum = 0 
        ans = 0 
        for i in range(len(nums)):
            sum += nums[i]
            if sum > 0 :
                ans += 1
        return ans

            
            